jll/L/libgeotiff_jll/Compat.toml

```
@@ -23,15 +23,25 @@ julia = "1.6.0-1"
 Libtiff_jll = "4.3.0-4.4"
 PROJ_jll = "900.100"
 
-["100.700.101-100"]
-Libtiff_jll = "4.5.1-4"
-
 ["100.700.101-100.700"]
 LibCURL_jll = ["7.73.0-7", "8.0.1-8"]
 PROJ_jll = "900.200.100-900.200"
 
-["100.701-100"]
+["100.700.101-100.701"]
+Libtiff_jll = "4.5.1-4"
+
+["100.701"]
 Artifacts = "1"
-LibCURL_jll = "7.73.0-8"
 Libdl = "1"
 PROJ_jll = "901.300.0-901"
+
+["100.701-100"]
+LibCURL_jll = "7.73.0-8"
+
+["100.702-100"]
+Artifacts = ["0.0.0", "1"]
+JpegTurbo_jll = "3.0.1-3"
+Libdl = ["0.0.0", "1"]
+Libtiff_jll = "4.7.0-4"
+PROJ_jll = "902.500.0-902"
+Zlib_jll = "1.3.0-1"
```

```suggestion
Zlib_jll = "1.2.12-1"
```

Already done in the build script: https://github.com/JuliaPackaging/Yggdrasil/pull/9589